src/ui/handler/scroll_zoom.js

```diff
@@ -356,6 +377,32 @@ class ScrollZoomHandler {
     reset() {
         this._active = false;
     }
+
+    _addScrollZoomBlocker() {
+        if (this._map && !this._container) {
+            this._container = DOM.create('div', 'mapboxgl-scroll-zoom-blocker', this._map._container);
+            let alertMessage = '';
+
+            if (/(Mac|iPad)/i.test(window.navigator.userAgent)) {
+                alertMessage = this._map._getUIString('ScrollZoomBlocker.CmdMessage');
+            } else {
+                alertMessage = this._map._getUIString('ScrollZoomBlocker.CtrlMessage');
+            }
+
+            const textNode = window.document.createTextNode(alertMessage);
```

Comment:
[A bit of googling](https://stackoverflow.com/questions/31643204/textnode-or-textcontent/31643816) suggests maybe this is equivalent in this case and involves just a bit less typing?

```suggestion
            this._container.textContent = alertMessage;
```